pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "simple-toml-settings"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
leave the version number as-is please, I generally update this manually just before release, and may add another PR :+1:

Reply:
No worries.  I'm new to contributing open source, but it is a pretty obvious rule to defer to the sensibilities and workflow of the maintainer. =-.)

Reply:
> No worries. I'm new to contributing open source, but it is a pretty obvious rule to defer to the sensibilities and workflow of the maintainer. =-.)

it's more that this will be a minor version bump, as you correctly did, and I may have a few more PR's to add before releasing it :smile: